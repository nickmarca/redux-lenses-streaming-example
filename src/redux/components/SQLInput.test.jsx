import React from 'react';
import { create } from 'react-test-renderer';
import SQLInput from './SQLInput';

describe('SQLInput', () => {
    it('matches the snapshot', () => {
        const sqlInput = create(<SQLInput />);

        expect(sqlInput.toJSON()).toMatchSnapshot();
    });
});
