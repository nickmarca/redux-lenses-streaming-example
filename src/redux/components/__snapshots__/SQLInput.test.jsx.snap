// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SQLInput matches the snapshot 1`] = `
<div
  className="panel-heading"
>
  <div
    className="field has-addons"
  >
    <p
      className="control is-expanded"
    >
      <textarea
        className="textarea is-small is-info"
        placeholder="SQLS"
        rows="3"
      />
    </p>
  </div>
</div>
`;
